app/api/schema/speakers_calls.py

```diff
@@ -32,9 +32,20 @@ def validate_date(self, data, original_data):
             if 'ends_at' not in data:
                 data['ends_at'] = speakers_calls.ends_at
 
+            if 'event_ends_at' not in data:
+                data['event_ends_at'] = speakers_calls.event.ends_at
+
         if data['starts_at'] >= data['ends_at']:
             raise UnprocessableEntity({'pointer': '/data/attributes/ends-at'}, "ends-at should be after starts-at")
 
+        if data['starts_at'] > data['event_ends_at']:
+            raise UnprocessableEntity({'pointer': '/data/attributes/starts-at'},
+                                      "speakers-call starts-at should be before event ends-at")
+
+        if data['ends_at'] > data['event_ends_at']:
+            raise UnprocessableEntity({'pointer': '/data/attributes/ends-at'},
+                                      "speakers-call ends-at should be before event ends-at")
```

Comment:
Shouldn't it be before starts-at?

Reply:
@iamareebjamal for CFS, I think starts-at should be used. Thanks for pointing out, I used the same condition as for tickets here :sweat_smile: 
Will be making the changes :)

Reply:
@iamareebjamal I've made the required changes. Can you please transfer this issue to server - https://github.com/fossasia/open-event-frontend/issues/2653 ?
Also, please review